public/js/products.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  'mozilla-concrete-brick': require('json!mozilla-concrete-brick'),
+  'mozilla-concrete-mortar': require('json!mozilla-concrete-mortar'),
```

Comment:
Filed https://github.com/mozilla/payments-ui/issues/100 to see if we can address this.